hello-world/nodejs-express-app/start-instance.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Sample code to start the given instance based on instance OCID and region.
+ * 
+ * Sample Output:
+ * (Prints the instance details)
+ */
+
+const common = require("oci-common");
+const core = require("oci-core");
+
+const adp = new common.ConfigFileAuthenticationDetailsProvider();
+
+const region = common.Region.US_ASHBURN_1;
```

Comment:
Same as above comments.

Reply:
Yes. its now taken from environment variable. However, this is the default value. It can be overridden via query parameter. You may check below code
```
        computeClient.region = region;
        if(req.query.region)
            computeClient.region = req.query.region;
```